src/style.css

```diff
@@ -0,0 +1,71 @@
+/* body {
+  box-sizing: border-box;
+} */
+
+.bigSearchBarForm {
+  margin: 20px auto;
+}
+
+.searchInput {
+  width: 100%;
+  box-shadow: 0 1px 8px, rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  outline: none;
+  
+}
+
+.bigSearchBarInput {
+  font-size: 16px;
+  height: 70px;
+  border-radius: 4px;
+  background: url("images/Search.svg") no-repeat 12px center / 19px;
+  padding-left: 62px;
+
+}
+
+.smallSearchBarInput {
+  font-size: 14px;
+  height: 42px;
+  border-radius: 3px;
+  background: url("images/Search.svg") no-repeat center / 11px;
+  background-position: 12px 14px;
+  padding: 10px 33px;
+
+}
+
+.searchInput:hover {
+  box-shadow: 0 3px 8px, rgba(61, 78, 97, 0.2);
+}
+
+.searchInput::placeholder {
+  font-weight: 300;
+}
+
+.bigSearchBarInput:focus {
+  background: url("images/Search.svg") no-repeat 12px center / 19px, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.smallSearchBarInput:focus {
+  background: url("images/Search.svg") no-repeat 12px center / 11px, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("fonts/Avenir-Heavy.ttf");
```

Comment:
Also you should set the same font family, but different font-weights so you can use different fonts on different font weights, e.g. 900 for Heavy, 300 for Book